### ISSUE TITLE
pre-trained model

### ISSUE THREAD

**Issue by qintaoxmu** [yangwenbo99/content-degradation-disentanglement#5]
Hello Author, I encountered a NaN gradient propagation error while training your model. I noticed that a pre-trained model is used in your configuration file. May I ask where this pre-trained model comes from?

**Comment by yangwenbo99** [comment]
Hi, please find the pretrained weights at [IVC lab's server](https://ivc.uwaterloo.ca/projects/content-degradation-disentanglement/weights/)
